templates/statefulset-zabbix-proxy.yaml

```diff
@@ -44,7 +44,11 @@ spec:
 {{- toYaml .Values.zabbixagent.resources | nindent 12 }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
+          {{- if .Values.zabbixagent_image.tag }}
+          image: "{{ .Values.zabbixagent.image.repository }}:{{ .Values.zabbixagent_image.tag }}"
```

Comment:
Hi @scoopex!

Can you change

`image: "{{ .Values.zabbixagent.image.repository }}:{{ .Values.zabbixagent_image.tag }}"`

by

`image: "{{ .Values.zabbixagent.image.repository }}:{{ .Values.zabbixagent.image.tag }}"`

?

Reply:
Sure, done :-)